### PR TITLE
Add packaging for ruby-foreman-azure-rm plugin (2.2.0)

### PR DESCRIPTION
**Pull request by eLvErDe**
Hello,

Generated deb package works and install cleanly on a Debian Buster and Azure showed up in compute resources so I guess it should work now !

Regards, Adam.

### CODE REVIEW

**Comment by ekohl** [comment]
ok to test

File: plugins/ruby-foreman-azure-rm/foreman_azure_rm/foreman_azure_rm.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
[the remark this answers is not in the document]
Comment (eLvErDe) [reply plugins/ruby-foreman-azure-rm/foreman_azure_rm/foreman_azure_rm.json:1]: That's indeed some cruft, I'm not sure why it ended up in the PR

File: plugins/ruby-foreman-azure-rm/debian/rules [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/make -f
+
+PLUGIN = foreman_azure_rm
+
+cache:
```
[the remark this answers is not in the document]
Comment (eLvErDe) [reply plugins/ruby-foreman-azure-rm/debian/rules:5]: Yes but that's a workaround to be able to run it outside the CI, I implemented the whole downloading inside debian/rules so I could build the package locally, imho it won't change anything on CI

**Comment by evgeni** [comment]
@eLvErDe can you please name the branch `deb/develop-<something>` something like `deb/develop-azure-rm`, otherwise Jenkins is confused :/

**Pull request closed without merging**